File: PS/algo/src/BOJ/Bronze/BOJ2869.java
package BOJ.Bronze;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class BOJ2869 {//풀이중
    public static void main(String[] args) throws IOException {
        BufferedReader br =new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int a =Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());
        int v = Integer.parseInt(st.nextToken());

        float answer = (float) (v-b)/(a-b);
        if(answer%1!=0){
            System.out.println((int) (answer+1));
        }else{
            System.out.println((int)answer);
        }

    }
}

File: algo/src/BOJ/Bronze/BOJ2869.kt
package BOJ.bronze

import java.io.BufferedReader

fun main()= with(BufferedReader(System.`in`.bufferedReader())) {
    val (a,b,v) =readLine().split(" ").map { it.toInt() }
    val n = a-b
    val m =v -a
    if(m%n==0) println(m/n+1)
    else println(m/n+2)
}